kas/opt/5.x-kernel.yml

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) Siemens AG, 2020
+#
+# This file is subject to the terms and conditions of the MIT License.  See
+# COPYING.MIT file in the top-level directory.
+#
+
+header:
+  version: 10
+
+local_conf_header:
+  ti-kernel-versions: |
+    PREFERRED_VERSION_linux-iot2050 = "5.4.%"
```

Comment:
Better to enclose this into SOC_VERSION, so that we can use an unified option yml file for both u-boot and kernel.